src/test/scala/latis/service/hapi/ToHapiTimeSpec.scala

```diff
@@ -149,7 +149,7 @@ class ToHapiTimeSpec extends AnyFlatSpec{
     val disp = Scalar.fromMetadata(Metadata("id"->"displacement", "type"->"int", "units"->"meters")).value
     val model = Function.from(time, disp).value
     val data = new SeqFunction(Seq(
-      Sample(time.convertValue("946688461001").toSeq, RangeData(1)),
+      Sample(DomainData(946688461001L), RangeData(1)),
```

Comment:
Note that the `type` in the metadata is `double` so I would use `946688461001.0` or `946688461001D` here.